packages/manager/apps/dedicated/client/app/billing/payment/method/routing.js

```diff
@@ -66,6 +66,8 @@ export default /* @ngInject */ (
           });
         }
       },
+
+      user: /* @ngInject */ (User) => User.getUser(),
```

Comment:
Not already injected in app root state ?

Reply:
You're right! We can user `currentUser`.

I will handle your suggestion into a separate Pull Request.